docusaurus.config.ts

```diff
@@ -20,6 +20,7 @@ const config: Config = {
   organizationName: 'fictionlab', // Usually your GitHub org/user name.
   projectName: 'fictionlab.github.io', // Usually your repo name.
   deploymentBranch: 'gh-pages',
+  trailingSlash: false,
```

Comment:
What's the reasoning behind this change?

Reply:
Right now all our pages have trailing slashes at the end, but sitemap has URLs without them. When Google crawls the site, it is redirected to site with trailing slash and the bot is not happy about it (refuse to index it).